test/controllers/ht_registrations_controller_test.rb

```diff
@@ -72,6 +72,22 @@ def setup
     assert_match(edit_btn, @response.body)
     assert_match(del_btn, @response.body)
   end
+
+  test "received registration shows WHOIS info" do
```

Comment:
This hits an external service, so we might want to limit when we run this, similarly to the HTML validation